infra/relayer/src/purse.js

```diff
@@ -438,23 +451,26 @@ class Purse {
    * @param txHash {string} - The transaction hash
    * @param rawTx {string} - The raw transaction
    */
-  async addPending(txHash, txObj, rawTx) {
+  async addPending(txHash, txObj, rawTx, sender = null) {
     this.pendingTransactions[txHash] = rawTx
 
     const to = txObj.to ? this.web3.utils.toChecksumAddress(txObj.to) : txObj.to
 
     // Store the tx object for debugging and in case we need to re-sign later
-    this.transactionObjects[txHash] = {
-      ...txObj,
-      to
+    this.transactionMeta[txHash] = {
+      originalSender: sender,
+      txObj: {
+        ...txObj,
+        to
+      }
     }
 
     if (this.rclient && this.rclient.connected) {
       await this.rclient.saddAsync(`${REDIS_PENDING_KEY}`, txHash)
       await this.rclient.setAsync(`${REDIS_PENDING_PREFIX}${txHash}`, rawTx)
       await this.rclient.setAsync(
         `${REDIS_PENDING_TX_PREFIX}${txHash}`,
-        JSON.stringify(txObj)
+        JSON.stringify(this.transactionMeta[txHash])
```

Comment:
I thought it was a little extreme to support both in this case with low load.  But since you brought it up, I'll add support for loading the old format.